src/nupic/encoders/random_distributed_scalar.py

```diff
@@ -152,9 +154,6 @@ def __init__(self, resolution, w=21, n=400, name=None, offset=None,
     else:
       self.name = "[%s]" % (self.resolution)
 
-    if self.verbosity > 0:
```

Comment:
lets keep the "verbosity" logic. Now with the __str__ you can use `print(self)` instead of dump()